lib/ruby_lsp/listeners/document_highlight.rb

```diff
@@ -180,7 +180,8 @@ def initialize(response_builder, target, parent, dispatcher)
       def on_call_node_enter(node)
         return unless matches?(node, [Prism::CallNode, Prism::DefNode])
 
-        add_highlight(Constant::DocumentHighlightKind::READ, node.location)
+        # TODO: understand why `message_loc` is nilable here
```

Comment:
If you have `foo.`, it's a call node, but there's no message yet. We need to return early if `message_loc` is `nil`.